dynamic-rapidpro-workspace-settings/test/index.spec.js

```diff
@@ -90,17 +90,30 @@ describe(`rapidpro action test suite`, () => {
   });
 
   it(`should be false if invalid flows data is given`, async () => {
-    let isValid = utils.isValidFlows({});
+    let isValid = utils.isValidObject({});
     expect(isValid).to.be.false;
```

Comment:
expect(utils.isValidObject({})).to.be.false;

You don't need the intermediate step to hold a value

Reply:
True.